backend/src/main/java/com/bakdata/conquery/sql/conversion/cqelement/concept/CQConceptConverter.java

```diff
@@ -47,13 +52,13 @@ public Class<CQConcept> getConversionClass() {
 	}
 
 	@Override
-	public ConversionContext convert(CQConcept node, ConversionContext context) {
+	public ConversionContext convert(CQConcept cqConcept, ConversionContext context) {
```

Comment:
warum ist der CQConceptConverter (noch) nicht in CQConcept inlined?

Reply:
Wir hatten uns erstmal (noch) dagegen entschieden, weil dann relativ viel Code im CQConverter liegt. Aber da können wir ja auch nochmal drüber sprechen, ob wir auch die verbleibenden Converter mit in die jeweiligen Klassen ziehen wollen. @torbsto